.github/workflows/pythonbuild.yml

```diff
@@ -79,35 +80,45 @@ jobs:
           - flytekit-spark
           - flytekit-sqlalchemy
         exclude:
-        # flytekit-modin depends on ray which does not have a 3.10 wheel yet.
-        # Issue tracked in https://github.com/ray-project/ray/issues/19116.
-        - python-version: 3.10
-          plugin-names: "flytekit-modin"
+          # dbt is not supported on 3.10 yet.
+          # https://github.com/dbt-labs/dbt-core/issues/4562
+          - python-version: 3.10
+            plugin-names: "flytekit-dbt"
+          # flytekit-modin depends on ray which does not have a 3.10 wheel yet.
+          # Issue tracked in https://github.com/ray-project/ray/issues/19116.
+          - python-version: 3.10
+            plugin-names: "flytekit-modin"
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Cache pip
-      uses: actions/cache@v2
-      with:
-        # This path is specific to Ubuntu
-        path: ~/.cache/pip
-        # Look to see if there is a cache hit for the corresponding requirements files
-        key: ${{ format('{0}-pip-{1}', runner.os, hashFiles('dev-requirements.txt', format('plugins/{0}/requirements.txt', matrix.plugin-names ))) }}
-    - name: Install dependencies
-      run: |
-        make setup
-        cd plugins/${{ matrix.plugin-names }}
-        pip install -e .
-        if [ -f dev-requirements.txt ]; then pip install -r dev-requirements.txt; fi
-        pip install --no-deps -U https://github.com/flyteorg/flytekit/archive/${{ github.sha }}.zip#egg=flytekit
-        pip freeze
-    - name: Test with coverage
-      run: |
-        cd plugins/${{ matrix.plugin-names }}
-        FLYTE_SDK_USE_STRUCTURED_DATASET=TRUE coverage run -m pytest tests
+      - uses: actions/checkout@v2
+      - name: Set up postgres (linux)
+        if: runner.os == 'Linux' && matrix.plugin-names == 'flytekit-dbt'
+        shell: bash
+        run: |
+          sudo systemctl start postgresql.service
+          pg_isready
+      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
Is there a material change in any of the steps below?  The diff isn't rendering well given the whitespace changes.